setup.py

```diff
@@ -3,12 +3,26 @@
 
 import sys
 
+from distutils.version import LooseVersion
+
+# We require setuptools 30.3.0 or later for the configuration in setup.cfg to
+# work properly.
+import setuptools
+if LooseVersion(setuptools.__version__) < LooseVersion('30.3.0'):
```

Comment:
I just looked and it seems like 30.3 was released Dec 6, 2016.  So >2 years, which is probably a pretty safe bet (even the LTS would be compatible, although presumably we're not doing that anyway).